examples/nlp_example.py

```diff
@@ -127,21 +140,38 @@ def collate_fn(examples):
         model, optimizer, train_dataloader, eval_dataloader, lr_scheduler
     )
 
+    # Potentially load in the weights and states from a previous save
+    if args.resume_from_checkpoint:
```

Comment:
There is a difference between this argument being None or "". In the second case we want to resume from the last checkpoint ideally.